pkg/tournament/classic_division.go

```diff
@@ -54,23 +54,24 @@ func (t *ClassicDivision) SetDivisionControls(divisionControls *realtime.Divisio
 		return nil, err
 	}
 	log.Debug().Interface("game-req", divisionControls.GameRequest).Msg("divctrls-validated-game-request")
-
+	if len(t.Matrix) < int(divisionControls.MinimumPlacement) {
+		return nil, fmt.Errorf("minimum placement is larger than the number of players: %d", t.DivisionControls.MinimumPlacement)
```

Comment:
This would break the flow where you add players after you set the division controls